run-development.sh

```diff
@@ -21,22 +21,7 @@ cd /usr/src/app/
 cmp -s /app/package.json /usr/src/app/app/package.json
 CHANGE_IN_PACKAGE_JSON="$?"
 
-## Copy node_modules to temporary location so we can reuse them, this occurs when the mountend sources have a node_modules and/or on restart
-rm -Rf /tmp/node_modules
-if [ -d /usr/src/app/app/node_modules/ ]
-then
-    mv ./app/node_modules /tmp/node_modules
-fi
-## Remove app folder if exists
-rm -rf ./app
-mkdir ./app
-if [ -d /tmp/node_modules/ ]
-then
-    mv /tmp/node_modules ./app/;
-fi
-
-## Copy app folder and config folder (including node_modules so host node_modules win)
-cp -rf /app ./
+rsync --delete -a --exclude=node_modules /app/ app/
```

Comment:
Using `rsync` with the `--exclude=node_modules` changes behavior compared to before. Previously the `node_modules` from the mounted volume in `/app` were taken into account and combined with the `node_modules` from a previous build cycle. Now they're not anymore. It will always only use the `node_modules` from a previous build cycle.